src/commands/install.ts

```diff
@@ -200,7 +201,9 @@ export default class Install extends Command {
           cli.action.start(chalk.grey(`-- Installing ${service_name} as dependency of ${config.getName()}`), undefined, { stdout: true });
           config.addDependency(service_name, service_tag);
           ServiceConfigBuilder.saveToPath(node.service_path, config);
-          const dep_node = await dependency_manager.loadService(`${service_name}:${service_tag}`);
+
+          // Hack to remove loadService from dependency manager
+          const dep_node = await dependency_manager.loadServiceFromConfig(plainToClass(ServiceConfigV1, { extends: `${service_name}:${service_tag}` }));
```

Comment:
`plainToClass(...)` should probably be replaced with `ServiceConfigBuilder.buildFromJson(...)` so that we don't hardcode v1 spec builds in several places.

Reply:
This code is specifically for build grpc stubs - I'll change to buildFromJson, but should we be removing this cruft entirely?

Reply:
Yes, we should. No longer used for anything since we deprecated the SDKs.